numbat/src/decorator.rs

```diff
@@ -49,6 +49,19 @@ pub fn get_canonical_unit_name(unit_name: &str, decorators: &[Decorator]) -> Str
     unit_name.into()
 }
 
+pub fn get_canonical_accepts_prefix(decorators: &[Decorator]) -> AcceptsPrefix {
+    for decorator in decorators {
+        if let Decorator::Aliases(aliases) = decorator {
+            for (_alias, accepts_prefix) in aliases {
+                if accepts_prefix.map(|ap| ap.short).unwrap_or(false) {
+                    return accepts_prefix.unwrap();
+                }
```

Comment:
I feel like this bit could be cleaned up somehow, but idk

Reply:
Hm, right. It could potentially also be combined with `get_canonical_unit_name` to reduce the code duplication?

The `if` statement could be written as
```rs
                match accepts_prefix {
                    &Some(ap) if ap.short => {
                        return ap;
                    }
                    _ => {}
                }
```
which is maybe a bit clearer, but quite a bit longer :man_shrugging: